Project.toml

```diff
@@ -17,6 +17,7 @@ ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 TimerOutputs = "a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
still no documenter in Project.toml

Reply:
I have no idea why that always shows up in the file. I delete it every time you say it but then it pops up again. Like it gets autogenerated.